### PR TITLE
MozPhab: init at 0.1.74

### PR DESCRIPTION
**Pull request by raboof**
###### Motivation for this change

MozPhab is the recommended tool for submitting patches to Mozilla Firefox.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5777,6 +5777,11 @@
       fingerprint = "7573 56D7 79BB B888 773E  415E 736C CDF9 EF51 BD97";
     }];
   };
+  raboof = {
```
Comment (sikmir) [inline maintainers/maintainer-list.nix]: Adding yourself as a maintainer is preferred to be a separate commit.
Something like, `maintainers: add raboof`. Then apply the MozPhab commit.

File: pkgs/applications/misc/MozPhab/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ];
+
+  meta = with lib; {
+    description = "Phabricator CLI from Mozilla to support submission of a series of commits";
+    longDescription = ''
+      MozPhab is a custom command-line tool, moz-phab, which communicates to
+      Phabricator’s API, providing several conveniences, including support for
+      submitting series of commits.
+    '';
+    homepage = https://moz-conduit.readthedocs.io/en/latest/phabricator-user.html;
```
Comment (sikmir) [inline pkgs/applications/misc/MozPhab/default.nix]: Quote URL please.

**Comment by raboof** [comment]
@sikmir thanks for the review - updated accordingly!

**Review by sikmir — APPROVED** [review]

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Comment by raboof** [comment]
abandoning this for now until I want to do a mozilla contribution again :)

**Comment by kvark** [comment]
I'm interested to bring this back to life
